common/src/main/java/com/wynntils/mc/event/ScoreboardSetScoreEvent.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.mc.event;
+
+import net.minecraft.server.ServerScoreboard;
+import net.minecraftforge.eventbus.api.Event;
+import org.jetbrains.annotations.Nullable;
+
+public class ScoreboardSetScoreEvent extends Event {
+    private final String owner;
+
+    @Nullable
```

Comment:
I know IntelliJ likes to sprinkle those `@Nullable` around, and I just hate them. If we want to have some kind of static null safety checks, we need to use these rigorously. And the resulting cluttering of code is hardly worth it imho. So please kill them, kill them with 🔥  ;-)